### ISSUE TITLE
Error showing up on homepage

### ISSUE THREAD

**Issue by erscampbell** [postmatic/epoch#136]
Submitted on behalf of Sidney Shema:

I woke up today, logged in to my WordPress website (shotsbyshema.com) and found the following message at the top of the website homepage - Notice: Undefined index: options in /home2/mkazi/public_html/shotsbyshema.com/wp-content/plugins/epoch/classes/core.php on line 190

The same message appears on the website's 'About' page as well.

It seems to imply that something needs to be changed or rectified in the Epoch plugin code. I am not savvy with code so I don't know how to proceed here. Please advise on what to do so that the message can get off the website as it detracts from the website's look.

**Comment by Vernal** [comment]
Can you verify which version of Epoch you are using?

On Fri, Oct 30, 2015 at 12:29 PM, Elissa Campbell notifications@github.com
wrote:

> Submitted on behalf of Sidney Shema:
> 
> I woke up today, logged in to my WordPress website (shotsbyshema.com) and
> found the following message at the top of the website homepage - Notice:
> Undefined index: options in /home2/mkazi/public_html/
> shotsbyshema.com/wp-content/plugins/epoch/classes/core.php on line 190
> 
> The same message appears on the website's 'About' page as well.
> 
> It seems to imply that something needs to be changed or rectified in the
> Epoch plugin code. I am not savvy with code so I don't know how to proceed
> here. Please advise on what to do so that the message can get off the
> website as it detracts from the website's look.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/postmatic/epoch/issues/136.

## 

Jason G. Lemieux | Creative Director
Vernal Creative and Strategy
http://www.vernalcreative.com | _New Number_ - 802.821.4059

**Comment by Vernal** [comment]
Also, please try de-activating and re-activating Epoch and see if the error persists.

**Comment by timbakkum** [comment]
Hey,

I got the same problem, same error on the pages with comments. I just installed the plugin for the first time, but on the epoch admin page, no options are showing because the javascripts and css are not loading eventhough the request URLs are correct. 

I initially installed it through composer (using bedrock) and retried it again through the wordpress admin plugins page, but no success.. 

EDIT : paths were actually wrong in the end. Had my WP_HOME env variable setup incorrectly causing plugin_dir_url() to be wrong. Sorry I wasted anybody's time!
